.dockerignore

```diff
@@ -1,3 +1,4 @@
 deploy
 input
 work
+output*
```

Comment:
This is actually a ghost directory which is not used anymore in `v2`. It was changed to utilize `deploy` and `work`, the  `deploy` directory contains the final images while all the temporary files are store in `work` (which is cleaned when conversion completes or fails).

So this line should be dropped instead,

https://github.com/mendersoftware/mender-convert/blob/next/docker-mender-convert#L22

Reply:
Thanks. Fixed